contrib/gplate/geojson2json.ipynb

```diff
@@ -0,0 +1,108 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Geojson2json.py is a simple script that directly converts GeoJSON files extracted using the GPlates GUI to the World Builder-compatible JSON format. Currently, it supports continental plate geometries. \n",
+    "\n",
+    "The script reads the GeoJSON file, processes polygon features, and assigns default or provided values for \n",
+    "maximum depth and compositions. The output JSON file follows the World Builder structure, making it ready \n",
+    "for use in geodynamic modeling applications.\n",
+    "\n",
+    "Future improvements may integrate this script with GPlate2GWB for a more streamlined workflow.\n",
+    "\n",
+    "Usage:\n",
+    "    python geojson2json.py\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "\n",
+    "# Configuration defined by the user\n",
+    "geojson_file = \"reconstructed_400.00Ma.geojson\"  # Input GeoJSON file\n",
+    "output_file = \"world_builder_model.json\"         # Output JSON file\n",
```

Comment:
this will work with the world builder, but I think world builder files are usually ending with `.wb` would it make sense to set that as output file type?